python/test_file_generator.py

```diff
@@ -2654,32 +2569,165 @@ def v705_2(self):
+		"""Set Open-End Line of Credit to 0."""
 		name = "v707.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.open_end_credit = lar.open_end_credit.map(lambda x: random.choice(("0", "")))
+		lar.open_end_credit = "0"
 		print("writing {name}".format(name=name))
 		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
 
 	def v708(self):
-		"""Set Business or Commercial Purpose to 0 or blank."""
+		"""Set Business or Commercial Purpose to 0."""
 		name = "v708.txt"
 		path = self.validity_path
 		ts = self.ts_df.copy()
 		lar = self.lar_df.copy()
-		lar.business_purpose = lar.business_purpose.map(lambda x: random.choice(("0", "")))
+		lar.business_purpose = "0"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v709(self):
+		"""Set City and Zip Code to Exempt and Street Address to not Exempt"""
+		name = "v709.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.street_address = "1234 Hocus Potato Way"
+		lar.city = "Exempt"
+		lar.zip_code = "Exempt"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v710_1(self):
+		"""Set Applicant Credit Score to Exempt, and set Co-Applicant Credit Score,
+		Applicant Score Name, and Co-Applicant Score Name to 650, 1, and 1 
+		respectively."""
+		name = "v710_1.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.app_credit_score = "1111"
+		lar.co_app_credit_score = "650"
+		lar.app_score_name = "1"
+		lar.co_app_score_name = "1"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v710_2(self):
+		"""Set Applicant Credit Score to Exempt, and set
+		Applicant Score Name, Co-Applicant Score Name, Applicant Score Code 8
+		and Co-Applicant Score Code 8 to 8, 8, "New Scoring Model", and "New Scoring Model" 
+		respectively."""
+		name = "v710_2.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.app_credit_score = "1111"
+		lar.app_score_name = "8"
+		lar.co_app_score_name = "8"
+		lar.app_score_code_8 = "New Scoring Model"
+		lar.co_app_score_code_8 = "New Scoring Model"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v711(self):
+		"""Set Denial 1 to Exempt, and set
+		Denial 2, Denial 3, Denial 4
+		and Denial Code 9 to 2, 3, 4, 
+		respectively."""
+		name = "v711.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.denial_1 = "1111"
+		lar.denial_2 = "2"
+		lar.denial_3 = "3"
+		lar.denial_4 = "4"
+		lar.denial_code_9 = ""
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v712(self):
```

Comment:
To test both cases, we should either have two sub-edits or a file that checks each condition.